gdal/swig/python/samples/README

```diff
@@ -4,115 +4,136 @@ Python interface may be used in your applications.
-val_repl.py		Script to replace specified values from the input
-			raster file with the new ones. May be useful in cases
-			when you don't like value, used for NoData indication
-			and want replace it with other value. Input file
-			remains unchanged, results stored in other file.
+val_repl.py
+    Script to replace specified values from the input raster file with the new ones.
+    May be useful in cases when you don't like value, used for NoData indication and
+    want replace it with other value.
+    Input file remains unchanged, results stored in other file.
 
-vec_tr.py		Example of applying some algorithm to all the
-			geometries in the file, such as a fixed offset.
+vec_tr.py
+    Example of applying some algorithm to all the geometries in the file, such as a fixed offset.
 
-vec_tr_spat.py		Example of using Intersect() to filter based on
-			only those features that truly intersect a given
-			rectangle.  Easily extended to general polygon!
+vec_tr_spat.py
+    Example of using Intersect() to filter based on	only those features
+    that truly intersect a given rectangle.
+    Easily extended to general polygon!
 
-classify.py             Demonstrates using numpy for simple range based
-                        classification of an image.  This is only an example
-                        that has stuff hardcoded.
+classify.py
+    Demonstrates using numpy for simple range based classification of an image.
+    This is only an example that has stuff hardcoded.
 
-gdal_lut.py             Read a LUT from a text file, and apply it to an image.
-                        Sort of a '1 band' version of pct2rgb.py.
+gdal_lut.py
+    Read a LUT from a text file, and apply it to an image.
+    Sort of a '1 band' version of pct2rgb.py.
 
-magphase.py             Example script computing magnitude and phase images
-                        from a complex image.
+magphase.py
+    Example script computing magnitude and phase images from a complex image.
 
-hsv_merge.py            Merge greyscale image into RGB image as intensity
-                        in HSV space.
+hsv_merge.py
+    Merge greyscale image into RGB image as intensity in HSV space.
 
-val_at_coord.py         Outputs the value of the raster bands at a given
-                        (longitude, latitude) or (X, Y) location.
+val_at_coord.py
+    Outputs the value of the raster bands at a given (longitude, latitude) or (X, Y) location.
 
-gdal_ls.py              Display the list of files in a virtual directory,
-                        like /vsicurl or /vsizip
+gdal_ls.py
+    Display the list of files in a virtual directory, like /vsicurl or /vsizip
 
-gdal_cp.py              Copy a virtual file
+gdal_cp.py
+    Copy a virtual file
 
-crs2crs2grid.py 	A script to produce PROJ.4 grid shift files from HTDP program.
-			http://trac.osgeo.org/proj/wiki/HTDPGrids
+crs2crs2grid.py
+    A script to produce PROJ.4 grid shift files from HTDP program.
+	http://trac.osgeo.org/proj/wiki/HTDPGrids
```

Comment:
Tab instead of four spaces before the URL

Reply:
now fixed